test/end-to-end/utils/dbAssertions.js

```diff
@@ -1,110 +1,62 @@
-import Future from 'fibers/future';
-
 import db from '../../util/db';
 
-const checkThatTheOutcomeDataWasWrittenToDatabase = ({
-  id,
-  outcome,
-}) => db.select()
-.table('prisoner_assessments')
-.where('id', Number(id))
-.then((result) => {
+export async function checkThatTheOutcomeDataWasWrittenToDatabase({ id, outcome }) {
+  const result = await db.select().table('prisoner_assessments').where('id', Number(id));
+
   expect(result[0]).to.not.equal(
     undefined,
     `Did not get a result from database for id: ${id}`,
   );
   expect(result[0].outcome).to.eql(outcome);
-  return result[0];
-});
+}
 
-const checkThatRiskAssessmentDataWasWrittenToDatabase = ({
-  id,
-  riskAssessment,
-}) => db.select()
-    .table('prisoner_assessments')
-    .where('id', Number(id))
-    .then((result) => {
-      expect(result[0]).to.not.equal(
-        undefined,
-        `Did not get a result from database for id: ${id}`,
-      );
-      expect(JSON.parse(result[0].risk_assessment).viperScore).to.eql(riskAssessment.viperScore);
-      expect(JSON.parse(result[0].risk_assessment).outcome).to.eql(riskAssessment.outcome);
-      expect(JSON.parse(result[0].risk_assessment).reasons).to.eql(riskAssessment.reasons);
-      expect(JSON.parse(result[0].risk_assessment).questions).to.eql(riskAssessment.questions);
-      return result[0];
-    });
+export async function checkThatRiskAssessmentDataWasWrittenToDatabase({ id, riskAssessment }) {
+  const result = await db.select()
+  .table('prisoner_assessments')
+  .where('id', Number(id));
 
-const checkThatHealthAssessmentDataWasWrittenToDatabase = ({
-  id,
-  healthAssessment,
-}) => db.select()
+  expect(result[0]).to.not.equal(
+    undefined,
+    `Did not get a result from database for id: ${id}`,
+  );
+  expect(JSON.parse(result[0].risk_assessment).viperScore).to.eql(riskAssessment.viperScore);
```

Comment:
can we store `JSON.parse(result[0].risk_assessment)` in a variable ?

Reply:
Changed it to be like the heath assessment check which does the entire check in one go